pkgs/servers/sql/postgresql/ext/pg_background.nix

```diff
@@ -0,0 +1,26 @@
+{
+  fetchFromGitHub,
+  lib,
+  postgresql,
+  postgresqlBuildExtension,
+}:
+
+postgresqlBuildExtension (finalAttrs: {
+  pname = "pg_background";
+  version = "1.5";
+
+  src = fetchFromGitHub {
+    owner = "vibhorkum";
+    repo = "pg_background";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-9fW5wHdo9r5fLwU8zN2EEVSWxa+7q2qMjPpMo6iCavg=";
+  };
+
+  meta = {
+    description = "PRun PostgreSQL Commands in Background Workers";
+    homepage = "https://github.com/vibhorkum/pg_background";
+    maintainers = with lib.maintainers; [ wolfgangwalther ];
```

Comment:
You'll have to add yourself as a maintainer to the maintainers list in a first commit (same PR, before the pg_background commit) and then add yourself here.

See https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md#how-to-become-a-maintainer